pypit/arlris.py

```diff
@@ -97,22 +99,22 @@ def read_lris(raw_file, det=None, TRIM=False):
     ny = ymax - ymin + 1
 
     #; change size for binning...
-    nx = nx / xbin
-    ny = ny / ybin
+    nx = nx // xbin
+    ny = ny // ybin
 
     #; Update PRECOL and POSTPIX
-    precol = precol / xbin
-    postpix = postpix / xbin
+    precol = precol // xbin
+    postpix = postpix // xbin
 
     # Deal with detectors
     if det in [1,2]:
-        nx = nx / 2
-        n_ext = n_ext / 2
-        det_idx = np.arange(n_ext) + (det-1)*n_ext
+        nx = nx // 2
+        n_ext = n_ext // 2
+        det_idx = np.arange(n_ext).astype(int) + (det-1)*n_ext
```

Comment:
For line 113 and 117, we can use (I think) `np.arange(n_ext, dtype=np.int)` to save recasting